create-custom-map-style/README.md

```diff
@@ -0,0 +1,904 @@
+
+    ```bash
+
+    npx tessera -r $(pwd)/node_modules/tilelive-aws/tilelive-aws.js aws:///CreateCustomMapStyle-<YOUR ENVIRONMENT NAME>
+    ```
+
+1. You should see some output in sdtout like `Listening at http://0.0.0.0:8080`.
+
+1. Now go back to Maputnik in your browser and click "Data Sources" in the top navigation bar. There should not be any active data sources. If you see any be sure to delete them.
+
+ ![Data Sources](media/MaputnikDataSources.png)
+ *Data Sources*
+
+1. We're going to add a new data source as shown below.
+
+- `Source ID`: `amazon`
+- `Source Type`: `Vector (XYZ URLs)`
+- `1st Tile URL`: `http://localhost:8080/{z}/{x}/{y}@2x.png`
+- `Min Zoom`: `0`
+- `Max Zoom`: `22`
+
+ ![New Data Source](media/MaputnikNewSource.png)
+ *New Data Source*
+
+1. In addition to Tiles, the type of web map we will be styling also makes use of Glyphs and Sprites. We can't use Tessera to proxy the endpoints for these, but we can however generate a signed URL for each endpoint to stick in our style descriptor JSON file.
+
+  (Optionally, we can choose to cheat and skip this step by providing some out of the box endpoints built in to Maputnik. Sprites: `https://maputnik.github.io/osm-liberty/sprites/osm-liberty` and Glyphs: `https://api.maptiler.com/fonts/{fontstack}/{range}.pbf?key={key}`)
+
+1. If you want to download a copy of the Sprites and Glyphs your map uses we've included a helper script `downloadSpritesGlyphs.sh` :
+
+    ```bash
+    ./downloadSpritesGlyphs.sh CreateCustomMapStyle-<YOUR ENVIRONMENT NAME> Amplify-Default
+    ```
+  
+This should create a folder for `sprites` and a folder for `glyphs` with a number of options for each. [Sprites](https://docs.aws.amazon.com/cli/latest/reference/location/get-map-sprites.html) can be either .json or .png files with an optional @2x version. [Glyphs](https://docs.aws.amazon.com/cli/latest/reference/location/get-map-glyphs.html) are a combination of a font and a Unicode range.
+
+1. If you choose to download the map Glyphs and Sprites you'll have to serve them on a port that is not already occupied. From `tileserver/`:
+
+    ```bash
+    npx http-server . -p 8088
+    ```
+
+1. While we could start from scratch to style the map it is easier to demonstrate with a starter template taken from our map style. Click "Open" in the top navigation bar and under "Upload Style" click "Upload" and select the `example-style-descriptor.json` file included in this repo.
+
+1. This is the style descriptor that is provided by Amazon Location Service as part of your Esri Map. You can download this JSON file by making a signed request to the API endpoint, i.e.
+
+    ```bash
+
+    aws location get-map-style-descriptor --map-name CreateCustomMapStyle style-descriptor.json --profile <YOUR AWS PROFILE NAME>
+    ```
+
+ We recommend using a SigV4 request-signing library such as [aws4](https://github.com/mhart/aws4#example).
```

Comment:
This feels out of place--is this a relic from before you published `tilelive-aws`?

Reply:
yes, another good catch

Reply:
updated